spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/HqlQueryRenderer.java

```diff
@@ -1455,9 +1454,8 @@ public List<JpaQueryParsingToken> visitSimpleCaseExpression(HqlParser.SimpleCase
 		tokens.add(new JpaQueryParsingToken(ctx.CASE()));
 		tokens.addAll(visit(ctx.expressionOrPredicate(0)));
 
-		ctx.caseWhenExpressionClause().forEach(caseWhenExpressionClauseContext -> {
-			tokens.addAll(visit(caseWhenExpressionClauseContext));
-		});
+		ctx.caseWhenExpressionClause()
+				.forEach(caseWhenExpressionClauseContext -> tokens.addAll(visit(caseWhenExpressionClauseContext)));
 
```

Comment:
I am usually a fan of shortening things like this, but since this is an ANTLR Visitor implementation, I actually prefer to retain it the way it is. It makes it easy when grammar tweaks require me to add another `tokens.addAll(...)` inside that clause. And since this pattern is used A LOT in this class (and its counterpart), it becomes easier to maintain if I retain the pattern consistently everywhere.